site/snippets/templates/meet/events.php

```diff
@@ -17,7 +17,7 @@ class="btn btn--filled"
 			<a href="<?= $event->link() ?>" class="flex items-center justify-between bg-white p-3 rounded shadow">
 				<h3 class="font-bold"><?= $event->title() ?></h3>
 				<p class="color-gray-700">
-					<?= $event->date()->toDate('D j M, H:i') ?> <?= $event->timezone() ?>
+					<localized-datetime><?= $event->date() ?></localized-datetime>
```

Comment:
`Suggestion` to keep the printed human-readable date in the HTML (= when the progressive enhancement fails) more readable:

```suggestion
					<localized-datetime date="<?= $event->date() ?>"><?= $event->date()->toDate('D j M, H:i T') ?></localized-datetime>
```